multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -187,129 +207,136 @@ test.serial('lp deposits', async t => {
+    const recipientAddress = encodeAddressHook(settlementAccount, { EUD });
+    t.log('recipientAddress', recipientAddress);
 
-  const { address: userForwardingAddr } = nobleTools.queryForwardingAddress(
-    nobleAgoricChannelId,
-    recipientAddress,
-  );
-  t.log('got forwardingAddress', userForwardingAddr);
-
-  const mintAmount = 800_000n;
-
-  // TODO export CctpTxEvidence type
-  const evidence = harden({
-    blockHash:
-      '0x90d7343e04f8160892e94f02d6a9b9f255663ed0ac34caca98544c8143fee665',
-    blockNumber: 21037663n,
-    txHash: `0xc81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff3875527617${makeRandomDigits(makeRandomNumber(), 2n)}`,
-    tx: {
-      amount: mintAmount,
-      forwardingAddress: userForwardingAddr,
-    },
-    aux: {
-      forwardingChannel: nobleAgoricChannelId,
+    // register forwarding address on noble
+    const txRes = nobleTools.registerForwardingAcct(
+      nobleAgoricChannelId,
       recipientAddress,
-    },
-    chainId: 42161,
-  });
-
-  console.log('User initiates evm mint', evidence.txHash);
-
-  // submit evidences
-  await Promise.all(
-    oracleWds.map(makeDoOffer).map((doOffer, i) =>
-      doOffer({
-        id: `${Date.now()}-evm-evidence`,
-        invitationSpec: {
-          source: 'continuing',
-          previousOffer: toOracleOfferId(i),
-          invitationMakerName: 'SubmitEvidence',
-          invitationArgs: [evidence],
-        },
-        proposal: {},
-      }),
-    ),
-  );
-
-  const queryClient = makeQueryClient(
-    await useChain('osmosis').getRestEndpoint(),
-  );
+    );
+    t.is(txRes?.code, 0, 'registered forwarding account');
 
-  await t.notThrowsAsync(() =>
-    retryUntilCondition(
-      () => queryClient.queryBalance(EUD, usdcOnOsmosis),
-      ({ balance }) => !!balance?.amount && BigInt(balance.amount) < mintAmount,
-      `${EUD} advance available from fast-usdc`,
-      {
-        // this resolves quickly, so _decrease_ the interval so the timing is more apparent
-        retryIntervalMs: 500,
+    const { address: userForwardingAddr } = nobleTools.queryForwardingAddress(
+      nobleAgoricChannelId,
+      recipientAddress,
+    );
+    t.log('got forwardingAddress', userForwardingAddr);
+
+    // TODO export CctpTxEvidence type
+    const evidence = harden({
+      blockHash:
+        '0x90d7343e04f8160892e94f02d6a9b9f255663ed0ac34caca98544c8143fee665',
+      blockNumber: 21037663n,
+      txHash: `0xc81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff3875527617${makeRandomDigits(makeRandomNumber(), 2n)}`,
+      tx: {
+        amount: mintAmt,
+        forwardingAddress: userForwardingAddr,
       },
-    ),
-  );
+      aux: {
+        forwardingChannel: nobleAgoricChannelId,
+        recipientAddress,
+      },
+      chainId: 42161,
+    });
+
+    console.log('User initiates evm mint:', evidence.txHash);
+
+    // submit evidences
+    await Promise.all(
+      oracleWds.map(makeDoOffer).map((doOffer, i) =>
```

Comment:
would love to see...

```suggestion
      oracleOperators.map(op => op.submit(evidence));
```